docsite/rst/developing_plugins.rst

```diff
@@ -9,10 +9,26 @@ in ansible and were most convenient to be made pluggable.
 This section will explore these features, though they are generally not common in terms of things people would look to extend quite
 as often.
 
-.. _developing_connection_type_plugins:
+.. _developing_modules:
 
-Connection Type Plugins
------------------------
+Task Plugins (aka Modules)
+--------------------------
+
+This is the most common type of plugin people need and develop, so we have a dedicated page for this type. See :doc:`developing_modules`.
```

Comment:
Task plugins are the most common type of plugin people need and develop. To learn all about them, refer to: :doc:`developing_modules`